_posts/2024-01-08-windows-support-1.11.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Windows support in Crystal 1.11"
+author: HertzDevil
+summary: Playground, preliminary interpreter support, and better dynamic linking
+---
+
+It has been 6 months since we last reported on the [status of Windows support in Crystal 1.9](/2023/07/06/windows-support-1.9). Although there aren't as many changes in [1.10](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.10.0+is%3Aclosed+label%3Aplatform%3Awindows) and [1.11](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.11.0+is%3Aclosed+label%3Aplatform%3Awindows), we have nonetheless made some significant breakthroughs which will be described below.
+
+## Playground support
+
+The playground now works on Windows. For a long time, `Process#wait` was synchronous on Windows, which means other fibers would not get the chance to run until the process exits. The playground server communicates with the compiled program via multiple WebSocket messages, and a synchronous `Process#wait` would effectively block the server. This is no longer the case since [#13908], and you will be able to use `crystal play` just like on other systems.
+
+## Finishing DLL coverage
+
+Currently, the compiler is statically linked for two reasons: the C function `LibC.snprintf` is not available in the C runtime DLLs, and the compiler depends on LLVM's C++ API, which requires a static LLVM build tree.
```

Comment:
**polish:** Just some clarification that this statement does not apply to the compiler *in general*.
Should be clear from the context, but it wouldn't hurt to be a bit precise.

```suggestion
Currently, the compiler is statically linked on Windows for two reasons: the C function `LibC.snprintf` is not available in the C runtime DLLs, and the compiler depends on LLVM's C++ API, which requires a static LLVM build tree.
```